tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -74,10 +78,15 @@ public void run() {
         // expected since the master will shutdown at a certain time.
         while (true) {
           if (mOpType == 0) {
+            FileOutStream stream = null;
             try {
-              mFileSystem.createFile(new AlluxioURI(TEST_FILE_DIR + mSuccessNum)).close();
+              stream = mFileSystem.createFile(new AlluxioURI(TEST_FILE_DIR + mSuccessNum));
```

Comment:
on looking at this closer it's actually exactly equivalent to the original 
```java
mFileSystem.createFile(new AlluxioURI(TEST_FILE_DIR + mSuccessNum)).close();
```
If `createFile` throws an exception, we won't set `stream` anyway